### PR TITLE
Include target="blank" vulnerability fix in HTML coding standards #23

### LINKED ISSUE #23: Include target="blank" vulnerability fix in HTML coding standards
As mentioned in https://github.com/TEAMMATES/teammates/pull/6368, we should include the fix in our coding standards to prevent future occurrences.

### PR DESCRIPTION
**Pull request by li-kai**
Fixes #23.

### CODE REVIEW

File: codingStandards/CodingStandard-Html.md [hunk 1]
```diff
@@ -237,3 +237,33 @@
...
+    </tr>
+    <tr>
+      <td>
+        <pre lang="html">
+  &lt;a target="_blank" href="https://example.com"&gt;
+    Example.com
+  &lt;/a&gt;</pre>
+      </td>
+    </tr>
+  </table>
```
Comment (damithc) [inline codingStandards/CodingStandard-Html.md]: We can include the rationale too?

File: codingStandards/CodingStandard-Html.md [hunk 2]
```diff
@@ -237,3 +237,33 @@
       </td>
     </tr>
   </table>
+
+- Always remember to include `rel="noopener noreferrer"` when using `target="_blank"` in links. Leaving this out would enable attackers to get hold of `window.opener` object, which, for example, can used to redirect to a malicious link. [\[Source\]](https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c#.jlkn46vtk)
```
Comment (damithc) [inline codingStandards/CodingStandard-Html.md]: While you are at it, give the rationale as a block quote? We use block quotes for additional info that can be skipped.

File: codingStandards/CodingStandard-Html.md [hunk 3]
```diff
@@ -237,3 +237,34 @@
       </td>
     </tr>
   </table>
+
+- Always remember to include `rel="noopener noreferrer"` when using `target="_blank"` in links. 
+> Leaving this out would enable attackers to get hold of `window.opener` object, which, for example, can used to redirect to a malicious link. [\[Source\]](https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c#.jlkn46vtk)
```
Comment (acjh) [inline codingStandards/CodingStandard-Html.md]: - No need `for example`.
- Missing 'be' in `can used to`.
- Is it necessary to escape the square brackets?
- The anchor in the link should be cleaned up.

File: codingStandards/CodingStandard-Html.md [hunk 4]
```diff
@@ -237,3 +237,34 @@
       </td>
     </tr>
   </table>
+
+- Always remember to include `rel="noopener noreferrer"` when using `target="_blank"` in links. 
+> Leaving this out would enable attackers to get hold of `window.opener` object, which can be used to redirect to a malicious link. [Source](https://medium.com/@jitbit/target-blank-the-most-underestimated-vulnerability-ever-96e328301f4c)
```
Comment (acjh) [inline codingStandards/CodingStandard-Html.md]: I meant if it's possible to do `[[Source]](...)` instead of `[\[Source\]](...)`.

Comment (li-kai) [inline codingStandards/CodingStandard-Html.md]: @acjh No it is not possible to have the brackets without escaping. However, I've looked at other markdown files and all other links in the other documentations have no brackets, so I've just removed them all together.

Comment (damithc) [inline codingStandards/CodingStandard-Html.md]: Another possibility `[[source](link)]`

**Pull request merged**